docs/logs/faq.md

```diff
@@ -73,16 +70,13 @@ Use [Logagent](/logagent).  With Logagent you can then do: ```cat
 **What is data retention and how does it work?**
 
 Data retention mechanism is responsible for removing old log events.
-If a log is 15 days old and you have retention set to 7 days, that log
-will remain in our centralized logging management platform until the retention mechanism deletes it. For
+If you have retention set to 7 days, all log events that are received more than 7 days ago will be removed by retention mechanism. For
```

Comment:
If you have retention set to 7 days, all log events that are received more than 7 days ago will be removed by retention mechanism. For
=>
If you have retention set to 7 days, all log events that are received more than 7 days ago will be removed. For